draft-ietf-httpbis-retrofit.md

```diff
@@ -70,7 +70,22 @@ This specification defines how a selection of existing HTTP fields can be handle
 
 It does so using two techniques. {{compatible}} lists compatible fields -- those that can be handled as if they were Structured Fields due to the similarity of their defined syntax to that in Structured Fields. {{mapped}} lists mapped fields -- those whose syntax needs to be transformed into an underlying data model which is then mapped into that defined by Structured Fields.
 
-Note that while implementations can parse and serialise compatible fields as Structured Fields subject to the caveats in {{compatible}}, a sender cannot generate mapped fields from {{mapped}} and expect them to be understood and acted upon by the recipient without prior negotiation. This specification does not define such a mechanism.
+Note that this specification does not enable use of Retrofit Structured Fields in the HTTP protocol "on the wire" or in APIs; it only establishes handling for specific fields that might be used by such applications in the future.
```

Comment:
I've read this statement a few times and I don't understand what you are trying to say.  (I *know* what you are trying to say, but I'm having a hard time drawing a straight line from these specific words to that concept.)

I also find this to be a little unsatisfactory in isolation.  Without some sort of additional coordination, we potentially leave the ecosystem in a bit of an odd position with the publication of a document that essentially says "here is a thing you might do that might (or might not) bring you benefits, but might (or might not) come with compatibility costs, you'll have to work out whether deploying it works for you".  I'd like to think that we can do better in terms of being concrete about our guidance.

Reply:
I'm going to pull this text out as it isn't strictly related to #2499; if you feel strongly about this, open an issue?